api/src/paths/project/{projectId}/funding-sources/{pfsId}/delete.ts

```diff
@@ -32,13 +33,14 @@ DELETE.apiDoc = deleteFundingSourceApiDocObject(
 
 export function deleteFundingSource(): RequestHandler {
   return async (req, res) => {
-    defaultLog.debug({ label: 'Delete project funding source', message: 'params', req_params: req.params });
+    const projectId = Number(req.params.projectId);
+    const pfsId = Number(req.params.pfsId);
```

Comment:
Is this shorthand for Platform Service ID?

Reply:
It stands for `project funding source id`. Some of this code could use a bit of a refactor/cleanup still, but I was just touching the bits that directly related to my changes.